indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskMaster.java

```diff
@@ -170,7 +161,7 @@ public void stopBeingLeader()
       {
         giant.lock();
         try {
-          initialized = false;
```

Comment:
Doesn't removing this mean that we miss out on the state of initialization of this class itself?